Sources/SwiftSyntax/Syntax.swift

```diff
@@ -541,13 +541,35 @@ public extension SyntaxProtocol {
     return self.with(\.leadingTrivia, []).with(\.trailingTrivia, [])
   }
 
-  /// The description of this node without the leading trivia of the first token
-  /// in the node and the trailing trivia of the last token in the node.
+  /// A copy of this node with pieces that match `matching` removed from the
```

Comment:
`removed` sounds to me like you’re removing all not just the prefix/suffix.

```suggestion
  /// A copy of this node with pieces that match `matching` trimmed from the
```